build.sbt

```diff
@@ -70,12 +47,12 @@ def baseProject(module: String, majorMinorVersion: String) = Project(s"$module-p
     scalaVersion := "2.13.11",
     crossScalaVersions := Seq(scalaVersion.value, "2.12.18"),
     scalacOptions := Seq(
+        "-release:11",
```

Comment:
[`gha-scala-library-release-workflow`](https://github.com/guardian/gha-scala-library-release-workflow) is currently using Java 17 for compiling everything, so it's important to specify that you want Java 11-compatible bytecode if you're going to be using this library in projects running Java 11.